### PR TITLE
Html code generation tests: Geo map charts + Mapbox map charts

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**

### CODE REVIEW

File: tests/Plotly.NET.Tests/HtmlCodegen/MapboxMapCharts.fs [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+let baseLayerOnlyChart = 
+    let mb =
+        Mapbox.init(
+            Style = StyleParam.MapboxStyle.OpenStreetMap
+        )
+    Chart.PointMapbox([],[]) // deliberately empty chart to show the base map only
+    |> Chart.withMapbox mb // add the mapBox
+
+[<Tests>]
+let `` charts`` =
```
Comment (kMutagene) [inline tests/Plotly.NET.Tests/HtmlCodegen/MapboxMapCharts.fs]: ``Mapbox charts``?

File: tests/Plotly.NET.Tests/HtmlCodegen/GeoMapCharts.fs [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+            Resolution=StyleParam.GeoResolution.R50,
+            ShowCountries=true, 
+            CountryColor="RebeccaPurple"
+        )
+    Chart.PointGeo([])
+    |> Chart.withMap countryGeo
+    |> Chart.withMarginSize(0, 0, 0, 0)
+
+[<Tests>]
+let ``Geo vs mapbox charts`` =
```
Comment (kMutagene) [inline tests/Plotly.NET.Tests/HtmlCodegen/GeoMapCharts.fs]: ``Geo charts`` ? i know the 'vs' path comes from the docs page but i do not think it makes sense here

**Review by kMutagene — CHANGES_REQUESTED** [review]

File: tests/Plotly.NET.Tests/HtmlCodegen/GeoMapCharts.fs (hunk 2 above)
Comment (WhiteBlackGoose) [reply tests/Plotly.NET.Tests/HtmlCodegen/GeoMapCharts.fs]: Noted. I still have no idea what `vs` means in this context, but okay, will remove

File: tests/Plotly.NET.Tests/HtmlCodegen/MapboxMapCharts.fs (hunk 1 above)
Comment (WhiteBlackGoose) [reply tests/Plotly.NET.Tests/HtmlCodegen/MapboxMapCharts.fs]: My bad!

File: tests/Plotly.NET.Tests/HtmlCodegen/GeoMapCharts.fs (hunk 2 above)
Comment (kMutagene) [reply tests/Plotly.NET.Tests/HtmlCodegen/GeoMapCharts.fs]: versus (both docs articles demonstrate the differences between Geo and MapBox maps and link each other)

**Review by kMutagene — APPROVED** [review]

**Pull request merged**